srcpkgs/rsnapshot/template

```diff
@@ -11,7 +11,7 @@ maintainer="Orphaned <orphan@voidlinux.org>"
 license="GPL-2.0-only"
 homepage="http://www.rsnapshot.org"
 distfiles="http://www.rsnapshot.org/downloads/${pkgname}-${version}.tar.gz"
```

Comment:
```suggestion
homepage="https://rsnapshot.org"
distfiles="https://rsnapshot.org/downloads/rsnapshot-${version}.tar.gz"
```

Reply:
I've added the changelog and updated homepage and distfiles to use https.